man/man8/zfs-list.8

```diff
@@ -30,12 +29,13 @@
 .\" Copyright 2018 Nexenta Systems, Inc.
 .\" Copyright 2019 Joyent, Inc.
 .\"
-.Dd June 30, 2019
+.Dd May 27, 2021
 .Dt ZFS-LIST 8
 .Os
+.
 .Sh NAME
 .Nm zfs-list
-.Nd Lists the property information for the given datasets in tabular form.
+.Nd tabulate properties for ZFS datasets
```

Comment:
I think this should use the verb "list" here, since it is `zfs list`.

"list properties for ZFS datasets"

Reply:
I wanted to avoid essentially just repeating the name bit, but sure